apps/docs/typedoc.config.mjs

```diff
@@ -12,14 +12,19 @@
 
 /** @type {Partial<import("typedoc").TypeDocOptions>} */
 export default {
-  entryPointStrategy: 'packages',
+  customCss: './custom.css',
+  disableSources: true,
   entryPoints: ['../../packages/*'],
+  entryPointStrategy: 'packages',
+  githubPages: false, // false === do not emit the file `.nojekyll`
+  name: 'Accelint Development Toolkit (DevTK)',
+  navigationLinks: {
+    // biome-ignore lint/style/useNamingConvention: <explanation>
```

Comment:
we could probably ignore naming convention for config files. if we leave the comment, let's add a real explanation

Reply:
I couldn't get Biome to ignore this so I put a small description.